scanamo/src/main/scala/org/scanamo/DynamoFormat.scala

```diff
@@ -58,7 +58,7 @@ trait DynamoFormat[T] {
   def xmap[U](r: T => Either[DynamoReadError, U], w: U => T): DynamoFormat[U] = DynamoFormat.xmap(r, w)(this)
 }
 
-object DynamoFormat extends PlatformSpecificFormat {
+object DynamoFormat extends PlatformSpecificFormat with FormatDerivation {
```

Comment:
Companion object with `derived` method enables `case class Foo(...) derives DynamoFormat` syntax in Scala 3.

Reply:
In scala 3, you can write

```scala
case class Foo(id:String, name: String) derives DynamoFormat
```